pwndbg/commands/search.py

```diff
@@ -50,7 +51,7 @@ def print_search_hit(address):
 parser = argparse.ArgumentParser(description='''
 Search memory for byte sequences, strings, pointers, and integer values
 ''')
-parser.add_argument('-t', '--type', choices=['byte','short','dword','qword','pointer','string','bytes'],
+parser.add_argument('-t', '--type', choices=['byte','word','dword','qword','pointer','string','bytes'],
```

Comment:
Please put `short` back? Thanks!  `word` is ambiguous, which is why `short` was explicitly chosen.